packages/trader/src/Modules/Trading/Containers/purchase.tsx

```diff
@@ -124,7 +125,7 @@ const Purchase = observer(({ is_market_closed }) => {
             />
         );
     }
-    return components;
+    return components as unknown as JSX.Element;
```

Comment:
@farzin-deriv I asked @kate-deriv to remove React.Fragment. We had issues in the past because of this when styles broke for components because React.Fragment tag was added just to satisfy TS. The current functionality is valid. If TS doesn't know that a component can return a node list, it's not our problem.

Reply:
Yes, it won't create errors, but isn't it the same as React.Fragment? I mean in case of some issue with style ([link](https://github.com/kate-deriv/deriv-app/pull/1#discussion_r1233973365))

Reply:
@farzin-deriv I've DM'd you 🙂